src/pages/Orgs/pages/OrgsId/pages/OrgRoot/components/ProofFieldForm.tsx

```diff
@@ -58,22 +72,28 @@ export default function ProofField({ proof }: Props) {
   ) : isSchemaEmpty || !formattedFieldKey ? (
     <></>
   ) : (
-    <Stack spacing={2}>
+    <form
+      onSubmit={e => {
+        e.preventDefault()
+        reload()
+      }}
+    >
       <UiTextField
         value={value}
         label={formattedFieldKey}
```

Comment:
how does it looks like?
may we try 
```ts
import lowerCase from 'lodash/lowerCase'
import upperFirst from 'lodash/upperFirst'

upperFirst(lowerCase('yopta_Kek_Cheburek'))
upperFirst(lowerCase('yoptaKekCheburek'))
```

<img width="264" alt="image" src="https://github.com/rarimo/rarime-app/assets/39623143/61d0716d-bf3e-4bc7-9b63-ce2a19c79631">

Reply:
it uses upperFirst for each word:
https://www.geeksforgeeks.org/lodash-_-startcase-method/

'age' -> 'Age'
'country_code' -> 'Country Code'
'firstName' -> 'First Name'